specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2021-10-01-preview/streamingjobs.json

```diff
@@ -571,6 +571,51 @@
         },
         "x-ms-long-running-operation": true
       }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.StreamAnalytics/streamingjobs/{jobName}/skus": {
+      "get": {
+        "tags": [
+          "StreamingJobs"
+        ],
+        "operationId": "StreamingJobs_GetSkus",
+        "description": "Gets a list of available SKUs about the specified streaming job.",
+        "x-ms-examples": {
+          "Get valid SKUs list for the specified streaming job": {
+            "$ref": "./examples/StreamingJob_GetSkus.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ApiVersionParameter"
```

Comment:
>v1 [](http://example.com/codeflow?start=34&length=2)

consider using v3 definitions

Reply:
Will this definition changes affect current behavior?

Reply:
We have no v3 definitions in path "../../common/", this file is defined by ourselves.

Reply:
oh k sorry i misread as common types